packages/cli/src/builders.ts

```diff
@@ -169,12 +169,12 @@ const builders: SnapsCliBuilders = {
     default: true,
   },
 
-  typescript: {
-    alias: 'ts',
-    describe: 'Use TypeScript template',
-    type: 'boolean',
+  template: {
+    alias: 't',
+    describe: 'Specify which template to use (TypeScript or JavaScript)',
+    type: 'string',
     demandOption: false,
-    default: true,
+    default: 'typescript',
```

Comment:
We should add an enum that represents the templates. Same as we did for `TranspilationModes`, then we can add `choices: Object.values(TemplateType)`